src/PendingMove.java

```diff
@@ -0,0 +1,38 @@
+import java.util.Scanner;
+
+public class PendingMove {
+    private Game game;
+    private Boolean validMove;
+
+    PendingMove(Game game) {
+        this.game = game;
+
+        Scanner scanner = new Scanner(System.in);
```

Comment:
This code is reading **U**ser **I**nput, so it belongs in the UI. You even have the same acronym! `PendingMove` will need to have a method that allows it to be updated with that input, after the UI has parsed it.